examples/hyperparameter/hyperparameter.py

```diff
@@ -0,0 +1,80 @@
+from typing import List
+
+import orchpy
+import time
+import numpy as np
+
+@orchpy.distributed([tuple], [np.ndarray])
+def linreg_gd(XY):
+    X = XY[0]
+    Y = XY[1]
+    alpha = 0.0001
+    w = np.zeros((X.shape[1], 1))
+    a = X.T.dot(Y)
+    b = X.T.dot(X)
+    for i in range(75000):
```

Comment:
where does the number 75000 come from? Maybe give this a descriptive variable name